server/data/visitSchedulerApiClient.test.ts

```diff
@@ -103,6 +103,36 @@ describe('visitSchedulerApiClient', () => {
     })
   })
 
+  describe('addExcludedDate', () => {
+    it('should return prison including the excluded date', async () => {
+      const prison = TestData.prison({ excludeDates: ['2023-12-25', '2023-12-26'] })
+
+      fakeVisitSchedulerApi
+        .put(`/admin/prisons/prison/${prison.code}/exclude-date/add`)
```

Comment:
Haven't tried, but I'm not sure if the contents of the request body (`2023-12-26`) actually getting tested here 🤔 ?

See: https://github.com/nock/nock#specifying-request-body - and an example at https://github.com/ministryofjustice/book-a-prison-visit-staff-ui/blob/8c6fc456455bbc89765ccf17749ba75e6cc8fc64/server/data/orchestrationApiClient.test.ts#L96